src/GridTurbulence.cpp

```diff
@@ -41,7 +41,7 @@ std::vector<std::pair<int, GridPrecision> > gridPowerSpectrum(ref_ptr<VectorGrid
 		for (size_t iy = 0; iy < n; iy++) {
 			for (size_t iz = 0; iz < n; iz++) {
 				i = ix * n * n + iy * n + iz;
```

Comment:
When modifying this code, you should reduce scope of i and declare it inside the inner loop. The compiler will optimize this - resulting code may be even better,as the variable is not required to exist at all.